docs/proposals/formatted-output.md

```diff
@@ -334,13 +336,13 @@ $REGISTRY/$REPO@sha256:a3785f78ab8547ae2710c89e627783cfa7ee7824d3468cae6835c9f4e
 
 View an artifact's referrers manifest in pretty JSON output. The following fields should be outputted:
 
```

Comment:
Shouldn't you still include the info for the subject here? The `reference`, `mediaType`, `size`, `digest` fields of the subject manifest before the `referrers` field?

Reply:
Updated